spdx/parsers/jsonyamlxml.py

```diff
@@ -1181,11 +1185,31 @@ def parse_pkg_down_location(self, pkg_down_location):
         else:
             self.value_error("PKG_DOWN_LOC", pkg_down_location)
 
+    def parse_pkg_files_analyzed(self, pkg_files_analyzed):
+        """
+        Parse Package files analyzed
+        - pkg_files_analyzed: Python boolean or str/unicode
+        """
+        # Files Analyzed optional
+        if pkg_files_analyzed is None:
+            return
+        if isinstance(pkg_files_analyzed, six.string_types) or isinstance(pkg_files_analyzed, bool):
```

Comment:
according to the spec, https://spdx.github.io/spdx-spec/3-package-information/
filesAnalysed can only be a boolean.

So I would have stored the conversion to bool.

Also, we are removing six in #179 , so please don't use six for new code/